apps/trading/client-pages/markets/markets-page.tsx

```diff
@@ -13,58 +9,213 @@ import {
 } from '@vegaprotocol/environment';
 import { useT } from '../../lib/use-t';
 import { ErrorBoundary } from '../../components/error-boundary';
-import { MarketsSettings } from './markets-settings';
 import { usePageTitle } from '../../lib/hooks/use-page-title';
+import { Card } from '../../components/card';
+import { useDataProvider } from '@vegaprotocol/data-provider';
+import {
+  activeMarketsWithCandlesProvider,
+  calcCandleVolumePrice,
+  type MarketMaybeWithCandles,
+} from '@vegaprotocol/markets';
+import { useYesterday } from '@vegaprotocol/react-helpers';
+import { useEffect, useState } from 'react';
+import { Interval } from '@vegaprotocol/types';
+import { formatNumber } from '@vegaprotocol/utils';
+import { TopMarketList } from './top-market-list';
+import {
+  useNewListings,
+  useTVL,
+  useTopGainers,
+  useTotalVolume24hCandles,
+} from './use-markets-stats';
+import classNames from 'classnames';
+
+const POLLING_TIME = 2000;
 
 export const MarketsPage = () => {
+  const t = useT();
+  const yesterday = useYesterday();
+  const {
+    data: activeMarkets,
+    error,
+    reload,
+  } = useDataProvider({
+    dataProvider: activeMarketsWithCandlesProvider,
+    variables: {
+      since: new Date(yesterday).toISOString(),
+      interval: Interval.INTERVAL_I1H,
+    },
+  });
+  useEffect(() => {
```

Comment:
Are we not able to set a pollInterval on the activeMarketsDataProvider?

CC @bglownia